doc/modules/time_series.rst

```diff
@@ -27,6 +27,18 @@ Time-delay embedding
 
    time_series.TakensEmbedding
 
+.. autosummary::
+   :toctree: generated/time_series/embedding/
+   :template: function.rst
+
+   time_series.takens_embedding_optimal_parameters
+
+.. autosummary::
+   :toctree: generated/time_series/embedding/
+   :template: class.rst
+
+   time_series.SingleTakensEmbedding
```

Comment:
This creates two classes list and one function lists. Why not putting it after `time_series.TakensEmbedding`?

Reply:
Thanks, well spotted!

Reply:
Done.